pmm-app/src/pmm-inventory/Inventory.tsx

```diff
@@ -17,6 +17,7 @@ export const InventoryPanel = () => {
     <div id="inventory-wrapper">
       <TabsBar>
         {tabs.map((tab, index) => (
+          // @ts-ignore
```

Comment:
The build is failing with similar errors to this one because of "Property 'css' missing in type...". I see that it has to do with emotion because I tried to add `@emotion/core` to package.json and then import it and the error stopped, but maybe there's another way to fix it.

Reply:
@tiagomotasantos I don't get it: emotion is not imported here. Why would it fail?